.changeset/grumpy-years-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
```

Comment:
actually, on second thought, should this be `minor`? I think this would be my first ever non-patch PR :smile: 
```suggestion
'@sveltejs/kit': minor
```